src/elements/pill.svelte

```diff
@@ -1,45 +1,61 @@
-<svelte:options immutable tag='v-pill' />
-
-<script lang='ts'>
+<svelte:options immutable tag="v-pill" />
```

Comment:
@micheal-parks Fixed the spacing but it deleted your comment. Heads up The changes to pill.svelete and checkKeyboardEvent function in events.ts are changes given to me by Devin so Im not super opinionated about them